projection/core/src/multi-jvm/scala/com/lightbend/lagom/internal/projection/ProjectionRegistrySpec.scala

```diff
@@ -191,15 +183,16 @@ class ProjectionRegistrySpec
       val allButTag001   = tagNames.tail
       val coordinates001 = WorkerCoordinates(projectionName, tagNames.head)
 
-      val testProbe = registerProjection(projectionName, tagNames.toSet)
-
+      val testProbe = TestProbe()
       // only let tagName001 messages pass
       testProbe.ignoreMsg {
         case FakeStarting(`tagName001`) => false
         case FakeStopping(`tagName001`) => false
         case _                          => true
       }
```

Comment:
You than look to this code for hours and start to believe that you don't understand how pattern matching with stable identifiers work.